omod/src/main/java/org/openmrs/module/pihapps/page/controller/labs/LabOrdersPageController.java

```diff
@@ -1,113 +1,23 @@
 package org.openmrs.module.pihapps.page.controller.labs;
 
-import org.apache.commons.lang.BooleanUtils;
-import org.openmrs.Concept;
-import org.openmrs.Order;
 import org.openmrs.Patient;
-import org.openmrs.TestOrder;
-import org.openmrs.api.OrderService;
-import org.openmrs.api.db.hibernate.HibernateUtil;
 import org.openmrs.module.emrapi.patient.PatientDomainWrapper;
-import org.openmrs.module.pihapps.orders.LabOrderConfig;
 import org.openmrs.module.pihapps.PihAppsConfig;
 import org.openmrs.ui.framework.UiUtils;
 import org.openmrs.ui.framework.annotation.InjectBeans;
 import org.openmrs.ui.framework.annotation.SpringBean;
 import org.openmrs.ui.framework.page.PageModel;
-import org.openmrs.util.OpenmrsUtil;
 import org.springframework.web.bind.annotation.RequestParam;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.TreeSet;
-
 public class LabOrdersPageController {
 
     public void get(PageModel model, UiUtils ui,
                       @InjectBeans PatientDomainWrapper patientDomainWrapper,
                       @RequestParam(value = "patient") Patient patient,
-                      @RequestParam(value = "testConcept", required = false) Concept testConcept,
-                      @SpringBean("orderService") OrderService orderService,
                       @SpringBean("pihAppsConfig") PihAppsConfig pihAppsConfig) {
 
```

Comment:
With the exception of basic UI Framework stuff (patient header, breadcrumbs, etc) moved this entirely to JS via REST endpoints, similarly to the labWorkflow page.